hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -391,7 +391,7 @@ public String getOzoneManagerSnapshotUrl() throws IOException {
           omLeaderUrl = (policy.isHttpsEnabled() ?
               "https://" + info.getServiceAddress(Type.HTTPS) :
               "http://" + info.getServiceAddress(Type.HTTP)) +
-              OZONE_DB_CHECKPOINT_HTTP_ENDPOINT;
+              OZONE_DB_CHECKPOINT_HTTP_ENDPOINT_V2;
```

Comment:
This code is used by Recon and here it is assumed that during upgrade scenario, Recon is upgraded after OM.

Reply:
Same goes with the client, it's just that recon here is the client. If client (an OM follower) gets upgraded first, it won't be able to download the checkpoint unless server gets upgraded